packages/pnpmfile/test/index.ts

```diff
@@ -1,35 +1,24 @@
-import { requirePnpmfile } from '@pnpm/pnpmfile'
+import { requirePnpmfile, BadReadPackageHookError } from '@pnpm/pnpmfile'
```

Comment:
I need expose `BadReadPackageHookError` to have a more reliable expectation, `toThrow()` only is able to catch either an object or a message and I wanna verify the error code.

Reply:
Greate, this is much better